workflows/pipe-common/scripts/mount_storage.py

```diff
@@ -63,9 +65,9 @@ class MountStorageTask:
     def __init__(self, task):
         self.api = PipelineAPI(os.environ['API'], 'logs')
         self.task_name = task
-        available_mounters = [NFSMounter, S3Mounter, AzureMounter]
+        available_mounters = [NFSMounter, S3Mounter, AzureMounter, GCPMounter]
         self.mounters = {mounter.type(): mounter for mounter in available_mounters}
-        self.default_regions = {'AWS': 'us-east-1', 'AZURE': 'eastus'}
+        self.default_regions = {'AWS': 'us-east-1', 'AZURE': 'eastus', 'GCP': 'europe-west3-c'}
```

Comment:
I'm not sure that default region for `AZURE` and `GCP` shall be specified.

Reply:
seems that we don't need `default_regions` at all, since it used only in one method `_get_cloud_region` and this method is unused